### ISSUE TITLE
Argument of ERB.new is deprecated

### ISSUE THREAD

**Issue by larseggert** [cabo/kramdown-rfc2629#92]
I get a ton of messages like this now:
```
/Users/lars/Documents/Code/kramdown-rfc2629/lib/kramdown-rfc/refxml.rb:11: warning: Passing safe_level with the 2nd argument of ERB.new is deprecated. Do not use it, and specify other arguments as keyword arguments.
/Users/lars/Documents/Code/kramdown-rfc2629/lib/kramdown-rfc/refxml.rb:11: warning: Passing trim_mode with the 3rd argument of ERB.new is deprecated. Use keyword argument like ERB.new(str, trim_mode: ...) instead.
```

**Comment by cabo** [comment]
The keyword parameters for ERB.new were introduced with Ruby 2.4.
I was trying to keep up compatibility down to Ruby 2.3, as this is still the on-board Ruby on macOS 10.14 (Mojave) and 10.13 (High Sierra).  Many people stay on this revision because macOS 10.15 (Catalina) nuked 32-bit applications.

So it seems I need to do something like
https://github.com/rails/rails/pull/32170/files

Interestingly, I don't get these messages.

```
$ ruby --version
ruby 3.0.0p0 (2020-12-25 revision 95aff21468) [x86_64-darwin19]
```

**Comment by larseggert** [comment]
Same version
```
ruby 3.0.0p0 (2020-12-25 revision 95aff21468) [x86_64-darwin20]
```
but I do...

**Comment by larseggert** [comment]
Do a `gem update` and see if you get them then?

**Comment by cabo** [comment]
Ah.  

```
>> ERB.version
=> "erb.rb [2.2.0 $]"
```
➔
```
>> ERB.version
=> "2.2.2"
```

Yep, I now get them, too.

**Comment by cabo** [comment]
Fixed in 1.3.25: a41d8c4
